etrago/cluster/disaggregation.py

```diff
@@ -669,15 +710,21 @@ def solve_partial_network(
                         else ()
                     )
                     ws = weight.sum(axis=len(loc))
-                    for bus_id in filtered.index:
-                        values = clt * weight.loc[loc + (bus_id,)] / ws
-                        pn_t[s].insert(len(pn_t[s].columns), bus_id, values)
+                    new_columns = pd.DataFrame(
+                        {
+                            bus_id: clt * weight.loc[loc + (bus_id,)] / ws
+                            for bus_id in filtered.index
+                        }
+                    )
+                    pn_t[s].loc[:, new_columns.columns] = new_columns
 
     def transfer_results(self, *args, **kwargs):
-        kwargs["bustypes"] = ["generators", "storage_units"]
+        kwargs["bustypes"] = ["generators", "links", "storage_units", "stores"]
         kwargs["series"] = {
             "generators": {"p"},
```

Comment:
In case a pf was performed, the `q` should be also transferred for generators and storage_units. 
Could you add this?

Reply:
I'd only have one question: how would I check whether a PF was performed? Or can I transfer those "results" unconditionally? The thing is, that this PR deals with `"links"` and `"stores"`, so this bug was present before updating the disaggregation. That's why I moved this to a [separate issue][0] (#644).

[0]: https://github.com/openego/eTraGo/issues/644